dspace-api/src/main/java/org/dspace/app/util/Version.java

```diff
@@ -62,17 +62,13 @@ public static void main(String[] argv)
 
         // Is Discovery available?
         ConfigurationService config = DSpaceServicesFactory.getInstance().getConfigurationService();
-        String consumers = config.getPropertyAsType("event.dispatcher.default.consumers", ""); // Avoid null pointer
-        List<String> consumerList = Arrays.asList(consumers.split("\\s*,\\s*"));
-        if (consumerList.contains("discovery"))
-        {
-            System.out.println("     Discovery:  enabled.");
-        }
-
-        // Is Lucene search enabled?
-        if (consumerList.contains("search"))
-        {
-            System.out.println(" Lucene search:  enabled.");
+        String[] consumers = config.getArrayProperty("event.dispatcher.default.consumers");
+        for (String consumer : consumers) {
+            if (consumer.equals("discovery"))
+            {
+                System.out.println("     Discovery:  enabled.");
+                break;
+            }
```

Comment:
I wonder if it'd be worth adding an "else" here to also print out if Discovery is *not* enabled. That'd imply a flaw in someone's upgrade, as Discovery must be enabled for search/browse to function.